React/wbooks/src/app/components/ErrorContainer/index.js

```diff
@@ -1,20 +1,20 @@
 import React, { Component } from 'react';
 
-import errorIcon from '../../../../../../assets/error.png';
+import errorIcon from '../../assets/error.png';
 
 import styles from './styles.scss';
 
-class LoginError extends Component {
+class ErrorContainer extends Component {
   render() {
     return (
       <div className={styles.errorContainer}>
         <img src={errorIcon} className={styles.errorIcon} alt='errorIcon' />
         <div className={styles.messageContainer}>
-          <p className={styles.errorText}>{this.props.errors}</p>
+          <h1 className={styles.errorText}>{this.props.errors}</h1>
         </div>
       </div>
     );
   }
 }
 
-export default LoginError;
+export default ErrorContainer;
```

Comment:
Mmm if this error if just for login or signup, i think that `ErrorContainer` isn't a descriptible name. I'd change it